spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/loadbalancer/NacosLoadBalancer.java

```diff
@@ -143,15 +143,13 @@ private Response<ServiceInstance> getInstanceResponse(
 									.get("nacos.cluster");
 							return StringUtils.equals(cluster, clusterName);
 						}).collect(Collectors.toList());
-				if (!CollectionUtils.isEmpty(sameClusterInstances)) {
+				if (CollectionUtils.isEmpty(sameClusterInstances)) {
+					log.warn("Not filtering to the specified cluster instance node，name = {}, clusterName = {}, instance = {}",
+							serviceId, clusterName, serviceInstances);
+				} else {
 					instancesToChoose = sameClusterInstances;
```

Comment:
Perhaps a log.info level log should be added here to indicate that the call to the cluster instance has started. Any idea?

Reply:
✌️Good idea, I have submitted a new PR, please review it again.